### PR TITLE
Create components and structure

### PR DESCRIPTION
**Pull request by spaceplesiosaur**
#### What's this PR do?

This PR adds the HTML structure to the app, and sets up the appropriate imports to the .js files.  There is no functionality here, and minimal class names.  

Since App.js needs a dynamic component to render, we can't see anything yet.

#### Where should the reviewer start?

You can look at each of the .js files - they should all be pretty small.  Please note, I added classnames where I felt like I would need labels for later, but did not implement all of them as I wanted to discuss what naming convention to use for classes with you.

#### How should this be manually tested?

It doesn't need to be tested at this point - since App.js needs a dynamic component, we need to set up that functionality before we can see anything.  Once that's done we can see the structure - for this PR, just lmk if you like the JSX and what I need to add/remove.

#### Any background context you want to provide?

Just as a note, if you think my sections should be div's I'm for it - I'm just used to sections!

Again, I added classnames when I basically needed a label, but wanted to put the rest on after discussing what naming convention we want to use.

In MovieBox.js on line 14 I added <MovieCard /> as a placeholder since I figured we aren't ready to write the functionality to map over the card array yet.  Did the same thing in CharacerBox.js.

### CODE REVIEW

File: src/components/CharacterCard/CharacterCard.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <section className="characterPlanetInfo">
+        <img className="planetIcon"></img>
+        <p className="planetName"></p>
+        <p className="planetPopulation"></p>
+      </section>
+      <h4 className="characterMoviesHeader"></h4>
+      <ul>
+        {/* Movie list placeholder*/}
+      </ul>
+    </section>
```
Comment (RayRedGoose) [inline src/components/CharacterCard/CharacterCard.js]: Can we add header tag for wrapping character name, div as wrapper for home description?

File: src/components/LoginForm/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          <label></label>
+          <input></input>
+        </section>
+        <section className="triviaLevelButtons">
+          <input type="radio">Novice</input>
+          <input type="radio">Intermediate</input>
+          <input type="radio">Expert</input>
+        </section>
+        <button>Submit</button>
+      </form>
```
Comment (RayRedGoose) [inline src/components/LoginForm/LoginForm.js]: I guess It will be better to change naming convention to BEM, and remove unnecessary class names.

**Review by RayRedGoose — CHANGES_REQUESTED** [review]
Great job with JSX! I guess we need to add some changes and can merge it.

File: src/components/LoginForm/LoginForm.js (hunk 2 above)
Comment (spaceplesiosaur) [reply src/components/LoginForm/LoginForm.js]: Sounds good, i will fix that up!

File: src/components/CharacterCard/CharacterCard.js (hunk 1 above)
Comment (spaceplesiosaur) [reply src/components/CharacterCard/CharacterCard.js]: Sounds good, will do!

**Pull request merged**